PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -73,7 +76,7 @@
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
 _mAOD = (run2_miniAOD_94XFall17 | run2_miniAOD_80XLegacy)
 (pp_on_AA | _mAOD).toReplaceWith(slimmingTask,
-                                 slimmingTask.copyAndExclude([slimmedLowPtElectronsTask]))
+                                 slimmingTask.copyAndExclude([slimmedLowPtElectronsTask, slimmedDisplacedMuons, slimmedDisplacedMuonTrackExtras]))
```

Comment:
Some of the runTheMatrix tests run on top of already made AOD which doesn't have the input to run the displacedMuon sequences in PAT. As a consequence, the tests fail in the PAT step. For the ones that I spotted, I have excluded displacedMuon processing in the Era modifier associated with the test. You can see an example in this line where the displacedMuons are excluded in a similar way to lowPtElectrons.

It this the correct approach to follow and is it safe for upcoming eras? Thanks a lot!